docs/release-notes/.FSharp.Compiler.Service/9.0.100.md

```diff
@@ -22,5 +22,5 @@
 * Optimize metadata reading for type members and custom attributes. ([PR #17364](https://github.com/dotnet/fsharp/pull/17364))
 * Enforce `AttributeTargets` on unions. ([PR #17389](https://github.com/dotnet/fsharp/pull/17389))
 * Ensure that isinteractive multi-emit backing fields are not public. ([Issue #17439](https://github.com/dotnet/fsharp/issues/17438)), ([PR #17439](https://github.com/dotnet/fsharp/pull/17439))
-
+* Enable FSharp 9.0 Language Version ([Issue #17497](https://github.com/dotnet/fsharp/issues/17438)), [PR](https://github.com/dotnet/fsharp/pull/17500)))
```

Comment:
Hi
Not sure if I'm approaching this issue correctly, but the link points to #17438 while the display text is #17497.

Reply:
Thanks for noticing, fixing it [here](https://github.com/dotnet/fsharp/pull/17883).